### PR TITLE
Add toggle checkbox component

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
@nlisgo This is the initial build of the pattern. Just want an opinion before I continue.

File: source/_patterns/00-atoms/forms/checkbox.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "checkbox-toggle",
+  "uncheckedValue": "item one",
+  "checkedValue": "item two"
+}
```
Comment (nlisgo) [inline source/_patterns/00-atoms/forms/checkbox.json:5]: nitpick: newline needed at end of file

File: source/_patterns/00-atoms/forms/checkbox.mustache [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="checkbox">
+  <div for="{{ name }}" class="checkbox__label selected-model" aria-label="{{ uncheckedValue }}">{{ uncheckedValue }}</div>
+
+  <label class="toggle-checkbox">
+    <input type="checkbox" id="{{ name }}" class="type-of-model">
+    <span class="toggle-checkbox__slider round"></span>
+  </label>
+
+  <div for="{{ name }}" class="checkbox__label" aria-label="{{ checkedValue }}">{{ checkedValue }}</div>
+</div>
```
Comment (nlisgo) [inline source/_patterns/00-atoms/forms/checkbox.mustache:10]: nitpick: newline needed at end of file

File: source/_patterns/00-atoms/forms/checkbox.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    minLength: 1
+  name:
+    type: string
+    minLength: 1
+  uncheckedValue:
+    type: string
+    minLength: 1
+  checkedValue:
+    type: string
+    minLength: 1
```
Comment (nlisgo) [inline source/_patterns/00-atoms/forms/checkbox.yaml:15]: nitpick: newline needed at end of file

**Comment by nlisgo** [comment]
Looks good: http://s3.amazonaws.com/ci-pattern-library/992/patterns/00-atoms-forms-checkbox/00-atoms-forms-checkbox.html

**Comment by nlisgo** [comment]
Is there an opportunity to add the form element to a page. Or is the atom good enough?

**Comment by discodavey** [comment]
@just adding class names and amending one to match other form field elements in the pattern library.

**Pull request merged**